### ISSUE TITLE
Easier installation and management of multiple service controllers

### ISSUE THREAD

**Issue by MarcusNoble** [aws/aws-controllers-k8s#553]
**Is your feature request related to a problem?**
Currently if wanting to make use of many service controllers you need to install, manage and update each individually.

**Describe the solution you'd like**
It'd be useful to have someway of managing multiple service controllers together. I see a couple potential ways:
1. A Kubernetes operator that can be used to create the desired service controllers based on provided configuration
2. A helm chart that contains all service controller helm charts as dependencies that are enabled via a provided value

The latter seems like it might be the easiest and reuses all the existing charts available today.

**Comment by jaypipes** [comment]
Hi @MarcusNoble! Yep, this is DEFINITELY on my TODO list.

As for a potential solution, I don't necessarily think a K8s operator would be the best solution (partly due to the way we generate everything else in ACK).

As for a single Helm chart, I could go for that or a Bash script that would essentially loop over a set of requested service aliases and install Helm charts for all the requested services.

**Comment by MarcusNoble** [comment]
Yeah, I don't think the operator is the right approach here either.

A single helm chart encompasing all the other charts would make it super easy to perform updates as you only have 1 release to worry about.
